<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Matthew Schmitt</title>
    <link rel="stylesheet" href="/css/styles.css">
</head>
<body>
    <nav class="main-nav">
        <div class="nav-content-container">
            <a href="/" class="nav-name">Matthew Schmitt</a>
            <ul class="nav-links">
                <li><a href="/index.html">About Me</a></li>
                <li><a href="/research.html">Research</a></li>
                <!--<li><a href="/assets/documents/your-cv.pdf" target="_blank">CV</a></li>-->
                <li><a href="/misc.html">Miscellaneous</a></li>
            </ul>
        </div>
    </nav>
    <div class="main-stripe">

        <div class="project-header">
            <h2>Machine learning interpretable models of cell mechanics from protein images</h2>
        </div>

        <div class="project-content">
            <div class="project-media-column">
                <!-- Example image; replace with your own media -->
                <video src="/assets/videos/washout_video_11_cell_1[1292].mov" controls muted loop autoplay playsinline>
                </video>
            </div>
            <div class="project-text-column">
                <p>
                In this work we use deep neural networks to predict traction forces exerted by cells directly from protein microscopy images.
                We then go on to constrain these networks with an interpretable biomechanical model to reveal how the network uses the information from the protein distributions to make its predictions.
                </p>
                <p>This work was the result of a great collaboration with the  <a href="https://patrickoakeslab.com/">Oakes group (Loyola)</a> and the <a href="https://squishycell.uchicago.edu/">Gardel group (UChicago)</a> and wouldn't have been possible without my co-first author <a href="https://jcolen.github.io/">Jonathan Colen</a>. 
                </p>
                <p>The paper was published in <a href="https://www.cell.com/cell/fulltext/S0092-8674(23)01331-4">Cell (2024)</a>.</p>
            </div>
        </div>

        <div class="project-header">
            <h2>Information theory for data-driven model reduction in physics and biology</h2>
        </div>

        <div class="project-content">
            <div class="project-media-column">
                <!-- Example image; replace with your own media -->
                <video src="/assets/videos/IB Cover Letter Video 1 - Abbreviated combined.m4v" controls muted loop autoplay playsinline>
                </video>
            </div>
            <div class="project-text-column">
                <p>
                In physics, one frequently reduces the description of a complex system to a simpler model that captures its essential features. In the process, one intuitively understands that information is being thrown away. 
                In this work, we make this intuition precise by formulating model reduction as a compression using tools from information theory. 
                Our theoretical results even allow us to understand the latent variables discovered by certain deep neural networks when applied to dynamical systems.
                </p>
                <p>
                This manuscript is currently under review. A preprint can be found on <a href="https://arxiv.org/pdf/2312.06608">arXiv</a>.
            </div>
        </div>



        <div class="project-header">
            <h2>
            Data-driven discovery of functional guilds in complex microbial communities
            </h2>
        </div>

        <div class="project-content">
            <div class="project-media-column">
                <!-- Example image; replace with your own media -->
                <video src="/assets/videos/microbiome_clustering_teaser.m4v" controls muted loop autoplay playsinline>
                </video>
            </div>
            <div class="project-text-column">
                <p>
                In this ongoing work we tackle the problem of identifying functional groups (or "guilds") of species in complex communities directly from data. These "species" can be literal species of bacteria (for example as measured by 16S sequencing), or they can be genes, proteins, or any other entities that interact in a complex system.
                </p>
                <p>
                This is work done in collaboration with Kiseok Lee (now U Delaware) and Seppe Kuehn (UChicago). This is currently work in progress.
                </p>
            </div>
        </div>


        

        <div class="project-header">
            <h2>
            Geometry adaptation of protrusion and polarity dynamics in confined cell migration
            </h2>
        </div>

        <div class="project-content">
            <div class="project-media-column">
                <img src="/assets/photos/cell_protrusion_paper.png" alt="A descriptive alt text for your image">
            </div>
            <div class="project-text-column">
                <p>
                How do cells move through confining spaces? Their motion results from the coordinated interactions of protrusions, adhesions with the environment, and the cell nucleus, and an internal polarity. 
                In this work we combine data-driven inference with a mechanistic bottom-up approach to model the protrusion and polarity dynamics of cells migrating in confining environments.
                Our model reveals how a feedback loop leads to stereotypical cycles of protrusion-nucleus dynamics that drive the migration of the cell through constrictions. 
                </p>
                <p>
                This work was performed during my master's thesis work in Chase Broedersz's group (then at LMU, now at VU Amsterdam) and was led by then-PhD student David Br√ºckner (now at <a href="https://www.biozentrum.unibas.ch/research/research-groups/research-groups-a-z/overview/unit/research-group-david-brueckner">University of Basel</a>).
                The paper was published in <a href="https://journals.aps.org/prx/abstract/10.1103/PhysRevX.12.031041">Physical Review X (2022)</a>.
                </p>
            </div>
        </div>


    </div></body>
</html>